toolkit/tools/osmodifierapi/verity.go

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package osmodifierapi
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/microsoft/azurelinux/toolkit/tools/imagecustomizerapi"
+)
+
+var (
+	verityNameRegex = regexp.MustCompile("^[a-z]+$")
+)
+
+type Verity struct {
+	// ID is used to correlate `Verity` objects with `FileSystem` objects.
+	Id string `yaml:"id"`
+	// The name of the mapper block device.
+	// Must be 'root' for the rootfs (/) filesystem.
+	Name string `yaml:"name"`
+	// The ID of the 'Partition' to use as the data partition.
+	DataDeviceId string `yaml:"dataDeviceId"`
```

Comment:
Rename to `DataDevice` and `HashDevice`.

(The `Id` suffix refers to the `Id` property on `Partition` type.)